File: G/submissions/accepted/g.py
n = int(input())
ans = 0
l = 0
for i in range(n):
    s = input()
    if l + len(s) > 80:
        ans += 1
        l = 0
    l += len(s) + 1
if l > 0:
    ans += 1
print(ans)

File: G/submissions/accepted/g.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n;
    scanf("%d",&n);
    int ans=0,now=80;
    for(int i = 0;i<n;i++){
        char c[105];
        scanf("%s",c);
        int len=strlen(c);
        if(now+len+1>80)now=len,ans++;
        else now+=len+1;
    }
    printf("%d\n",ans);
}